app/models/vm_or_template.rb

```diff
@@ -1923,6 +1923,10 @@ def validate_supported
     {:available => true,   :message => nil}
   end
 
+  def validate_unsupported(message_prefix)
```

Comment:
To keep this consistent, this should really be in `vm_or_template/operations.rb` since `vm/operations.rb` has the Vm version of `validate_unsupported`.